### DISCUSSION TITLE
Rascal

### DISCUSSION THREAD

**Discussion by ENDERSTIXgaming** [Faboslav/friends-and-foes#246] [General]
how does the loot table for this work? because they appear to be giving illegal items? for example. when paired with the "Better Nether" mod which has an enchant called Obsidian Breaker. (after checking their wiki it appears this enchant is exclusive to one of their modded pickaxes) one of my friends told me they're getting tools with obsidian breaker from the rascal that aren't from that mod. which intrigues me cause I assumed you'd have made a loot table with set items but we're getting items with enchants that wouldn't be possible in survival otherwise. not necessarily a complaint but it's interesting imo

**Comment by Faboslav** [comment]
I guess, you can get these enchant in all of the loot tables, whenever the `minecraft:enchant_with_levels` is used, i am also using that one. Mentioned loot table is called `rewards/rascal_good_reward.json`, you can check it if wanted :).
